.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,110 @@
+      description: Any other information that might help diagnose and fix the bug
+      placeholder: "Environmental factors, recent changes, etc..."
+    validations:
+      required: false
+  - type: input
+    id: meme-upload
+    attributes:
+      label: Mandatory Meme Upload
+      description: Please upload a meme that expresses your feelings about this bug
+      placeholder: "Drag and drop your meme image here or click to upload"
+    validations:
+      required: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)